### ISSUE TITLE
Bi-directional information not imported properly with Migrate Packages

### ISSUE THREAD

**Issue by m9n** [pods-framework/pods#3256]
When I use the Migrate Packages component to export and import my pods, the Bi-directional information is lost from Relationship fields.  It's repeatable:

- Create two Pods with a bi-directional Relationship with each other.
- Use Migrate Packages to export your Pods.
- Use WP Reset (plugin) to empty your database and start from a clean install.
- Enable Pods, and enable the Migrate Package component.
- Import your Pods from the export file you created.
- On the bi-directional relationship field, you'll find the bi-directional information is lost.

**Workaround:** Just make sure to add back your bi-directional settings in each of the Pods once you've imported.  No biggie :)

**Comment by m9n** [comment]
Quick follow up - you only actually loose the bi-directional settings on one of the Pods.  I suspect it's because they're created in sequence, and the second Pod hasn't yet been created.

**Comment by sc0ttkclark** [comment]
Bidirectional field config is storing the related field ID (post ID), so it's possible it's not exactly as you're seeing or perhaps it's a lucky import :)

**Comment by pglewis** [comment]
Is there an issue to be extracted here?

**Comment by sc0ttkclark** [comment]
I don't think we can solve this one pre-migration to storing keys in wp_podsrel instead of IDs.

**Comment by sc0ttkclark** [comment]
Just a note, this issue has existed as long as Bi-directional relationships and Packages have co-existed. (Even back to 1.x)

**Comment by jimtrue** [comment]
Brainstorming... keep track of the bidirectional relationship fields upon 'migrating' and as a last step, re-touch all those bidirectional fields to force the connection?

**Comment by vpfaulkner** [comment]
@jimtrue is this still a bug? Is there any way to export/import with bi-directional information retained?

**Comment by jimtrue** [comment]
As @sc0ttkclark notes above this will not be able to be corrected until Fields Objects are a part of Pods. 

In most cases, honestly, I've not had issues with this and I do package migrations constantly as part of replicating bugs from GitHub. 

Typically once you visit one of the configurations it corrects itself in the other.
